test/Libraries/RevitIntegrationTests/LevelTests.cs

```diff
@@ -10,35 +11,37 @@ namespace RevitSystemTests
     [TestFixture]
     class LevelTests : SystemTest
     {
-        [Test]
+        [Test, Category("Failure")]
```

Comment:
I think we already have lot many test cases marked as Failure and need to Revisit them at once. So marked this as Failure and removed its assert for Inconclusive. As you can see, it is not just a addition of Failure category to the tests. I have made other change as well.